clients/include/testing_gemm_ex.hpp

```diff
@@ -250,7 +250,7 @@ void testing_gemm_ex(const Arguments& arg)
 
     bool nantest = rocblas_isnan(arg.beta);
```

Comment:
This may need to be changed to:

`bool nantest = rocblas_isnan(arg.beta) || rocblas_isnan(arg.betai);`

For non-complex types, `betai` is `0` and so `rocblas_isnan(arg.betai)` is `false`.

Reply:
understand. update it in commit "update complex change 2".